src/main/java/com/bankbazaar/test/entity/Department.java

```diff
@@ -0,0 +1,24 @@
+package com.bankbazaar.test.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Table(name = "Department")
+public class Department{
+    @Id
+    @GeneratedValue
+    @Column(name="department_id")
+    private int id;
+    @Column(name="department_name")
+    private String name;
+    @Column(name="employee_count")
```

Comment:
Ideally department should not have this. This should be derived from the employee table

Reply:
I have replaced employeeCount with departmentFloor and departmentPhone